src/queryResult/queryResultWebViewController.ts

```diff
@@ -170,6 +170,18 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
         this.registerRequestHandler("getWebviewLocation", async () => {
             return qr.QueryResultWebviewLocation.Panel;
         });
+        this.registerRequestHandler("getFontFamily", async () => {
+            console.log("getFontFamily");
+            return this._vscodeWrapper
+                .getConfiguration(Constants.extensionName)
+                .get(Constants.extConfigResultKeys[3]);
```

Comment:
indexing into an array for a specific value feels like a recipe for bugs when values change.  Can you update the `extConfigResultKeys` array to instead be a string-type enum?

```
export const extConfigResultKeys = [
    shortcuts = "shortcuts",
    ...
];
```